### PR TITLE
Radhika

### PR DESCRIPTION
**Pull request by RadhikaAgarwal**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: Radhika [hunk 1]
```diff
@@ -0,0 +1,5 @@
+  name: Radhika Agarwal
+  country: India
+  github: RadhikaAgarwal
+  image: https://github.com/RadhikaAgarwal/gci17.fossasia.org/commit/388bc55f78bf64d68bcb7ac1187c308dc332df6e
```
Comment (nikit19) [inline Radhika:4]: You have to write the name of the image here

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please make sure that the dimensions of the image is 240 x 240 pixels.
And please add your photo and details in correct files ie in the students.yml and image should be inside img/students folder

File: Radhika (hunk 1 above)
Comment (anshumanv) [inline Radhika:4]: Please change it to image's file name.

**Review by NaviRocker — CHANGES_REQUESTED** [review]
Please Make sure to finish this Soon.

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@RadhikaAgarwal  Please do the suggested changes 👍

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Status ?

**Comment by mariobehling** [comment]
Closing due to inactivity.

**Pull request closed without merging**